lib/csv_export.rb

```diff
@@ -70,4 +70,20 @@ def download_equipment_data
     download_zip([categories, models, items],
                  "EquipmentData_#{Time.zone.now.to_s(:number)}")
   end
+
+  # NOTE: this method depends on ActionController
+  def download_category_data(cat)
```

Comment:
So this basically does the same thing as `download_equipment_data` except it restricts the export to a single category? I feel like this is a lot of duplication in that case.